src/code42cli/sdk_client.py

```diff
@@ -30,18 +32,24 @@ def create_sdk(profile, is_debug_mode):
         )
         py42.settings.verify_ssl_certs = False
     password = profile.get_password()
-    return validate_connection(profile.authority_url, profile.username, password)
+    return validate_connection(profile.authority_url, profile.username, password, totp)
 
 
-def validate_connection(authority_url, username, password):
+def validate_connection(authority_url, username, password, totp):
     try:
-        return py42.sdk.from_local_account(authority_url, username, password)
+        return py42.sdk.from_local_account(authority_url, username, password, totp=totp)
     except ConnectionError as err:
         logger.log_error(str(err))
-        raise LoggedCLIError("Problem connecting to {}".format(authority_url))
+        raise LoggedCLIError(f"Problem connecting to {authority_url}.")
+    except Py42MFARequiredError:
+        totp = prompt("Multi-factor authentication required. Enter TOTP", type=int)
+        return validate_connection(authority_url, username, password, totp)
     except Py42UnauthorizedError as err:
         logger.log_error(str(err))
-        raise Code42CLIError("Invalid credentials for user {}".format(username))
+        if "INVALID_TIME_BASED_ONE_TIME_PASSWORD" in err.response.text:
```

Comment:
This is fine for now but this seems like a common enough scenario that we should have a custom py42 error for this